charts/datadog/templates/cluster-agent-confd-configmap.yaml

```diff
@@ -14,6 +14,9 @@ data:
 {{- end }}
 {{- if .Values.datadog.kubeStateMetricsCore.enabled }}
   kubernetes_state_core.yaml.default: |-
+{{- if .Values.datadog.kubeStateMetricsCore.useClusterCheckRunners }}
+    cluster_checks: true
```

Comment:
typo? Should be `cluster_check` right? https://github.com/DataDog/helm-charts/issues/460 @CharlyF

Reply:
yes it is a typo.

Reply:
fix https://github.com/DataDog/helm-charts/pull/462 (turned out to be more than just a type 😅 )